commit c775072ad5a93306183f83f4396268b1690bb392
Author: Brad King <brad.king@kitware.com>
Date:   Wed Apr 22 09:58:46 2015 -0400

    cmGeneratorExpressionEvaluationFile: Add missing sys/types.h include
    
    This is needed for 'mode_t', and on mingw-w64 in particular.  We added
    use of 'mode_t' in commit v3.2.0-rc1~421^2 (file(GENERATE): Use
    permissions of input file if present, 2014-07-22).

--- Source/cmGeneratorExpressionEvaluationFile.h
+++ Source/cmGeneratorExpressionEvaluationFile.h
@@ -13,6 +13,7 @@
 #define cmGeneratorExpressionEvaluationFile_h
 
 #include "cmStandardIncludes.h"
+#include <sys/types.h>
 #include <cmsys/auto_ptr.hxx>
 
 #include "cmGeneratorExpression.h"
